habitat/core/spaces.py

```diff
@@ -59,10 +63,16 @@ def sample(self):
         }
 
     def contains(self, x):
-        if not isinstance(x, dict) and {"action", "action_args"} not in x:
+        if not isinstance(x, dict):
+            return False
+        if not all(key in x for key in {"action", "action_args"}):
```

Comment:
```suggestion
        if not set(x.keys()).issubset({"action", "action_args"}):
```

Reply:
Actually, looking at the code bellow, it seems to assume that `x` has both "action" and "action_args" keys, so maybe this should just be `x.keys() == {"action", "action_args"}`?

CC @mathfac

Reply:
I agree, I think it makes sense to restrict it to the two keys.

Reply:
To nicely support empty space, it seems like `x["action_args"]` -> `x.get("action_args", None)` would be better.

Reply:
Good point, that would simplify empty space actions from `x = {"action": "move_forward", "action_args": None}` to `x = {"action": "move_forward"}`.

Reply:
I made this change.